etc/vim/vimrc

```diff
@@ -63,6 +63,7 @@ call neobundle#begin(g:bundledir)
 NeoBundleFetch 'Shougo/neobundle.vim'
 
 " Declare used bundles:
+NeoBundle 'axvr/org.vim'
```

Comment:
Again, please remove this (I believe this is part of the same Emacs/Doom commit as the other change).

Reply:
Removed.